crates/egui_plot/src/lib.rs

```diff
@@ -1393,6 +1398,27 @@ impl PlotUi {
         &self.response
     }
 
+    /// Scale the plot bounds around a position in screen coordinates. Can be useful for
```

Comment:
each docstring should start with a one-sentence summary, followed by a blank line, followed by more details. `cargo doc` will put the first line in a summary view, and if there are no blank lines, the entire huge thing is put into a summary! Check `cargo doc -p egui_plot` to see what I mean